src/dags/crowdmonitor.py

```diff
@@ -152,15 +149,15 @@ def copy_data_in_batch(fetch_iterator, periode="uur"):
 args = default_args.copy()
 
 with DAG(
-    dag_id,
+    DAG_ID,
     default_args=args,
     description="Crowd Monitor",
-    on_failure_callback=get_contact_point_on_failure_callback(dataset_id=dag_id),
+    on_failure_callback=get_contact_point_on_failure_callback(dataset_id=DAG_ID),
 ) as dag:
     create_temp_tables = PostgresOperator(
         task_id="create_temp_tables",
         sql=SQL_CREATE_TEMP_TABLE,
-        params=dict(base_table=table_id),
+        params={'base_table': TABLE_ID},
```

Comment:
Strange, these are single quotes? Default behaviour of black is to use double quotes, right?

Reply:
That's strange indeed. Black, as run from the pre-commit hook, should have picked that up. Will fix.

Reply:
It's even stranger. Can't find it in my version of `crowdmonitor`. It's old code.

Reply:
Ah, now I get it. You are reviewing things per commit (as I recommended). This must have been the result of not wanting to do too many cleanups in one and the same commit. I probably did get many `flake8` and `mypy` errors and decided to address those in other commits. Hence I commited this with `--no-verify`, which also bypassed the `black` run. Later commits resolved this. Sorry for the confusion.

Anyway it has been addressed.